draft-peabody-dispatch-new-uuid-format-03.txt

```diff
@@ -528,1005 +575,748 @@ Internet-Draft               new-uuid-format               February 2022
+   The monotonicity properties of an particular UUID generator SHOULD
+   stated in it's documentation.
 
-   When decoding or parsing a UUIDv7 value there are only two values to
-   be considered:
+5.6.  Global and Local Uniqueness
 
-   1.  The unix timestamp defined as unixts
+   Global uniqueness is impossible to guarantee without shared
+   knowledge.  I.e. two systems cannot come up with two numbers
+   completely independently without some possibility of collision UNLESS
+   they agree on some mechanism to ensure uniqueness ahead of time (e.g.
+   your numbers always end with ... and mine always end with... or
+   whatever)
 
-   2.  The sub-second precision values defined as subsec_a, subsec_b,
-       and subsec_seq_node
+   RFC4122 tried to use MAC address as shared knowledge - it sort of
+   worked but had problems (security issues with exposing MAC addresses
+   defined in Section 7, guarantee of actual global uniqueness is
+   questionable)
 
-   As detailed in Figure 2 the unix timestamp (unixts) is always the
-   first 36 bits of the UUIDv7 layout.
 
 
 
 
-Peabody & Davis           Expires 5 August 2022                [Page 15]
+Peabody & Davis          Expires 26 August 2022                [Page 15]
 
 Internet-Draft               new-uuid-format               February 2022
 
 
-   Similarly as per Figure 2, the sub-second precision values lie within
-   subsec_a, subsec_b, and subsec_seq_node which are all interpreted as
-   sub-second information after skipping over the version (ver) and
-   (var) bits.  These concatenated sub-second information bits are
-   interpreted in a way where most to least significant bits represent a
-   further division by two.  This is the same normal place notation used
-   to express fractional numbers, except in binary.  For example, in
-   decimal ".1" means one tenth, and ".01" means one hundredth.  In this
-   subsec field, a 1 means one half, 01 means one quarter, 001 is one
-   eighth, etc.  This scheme can work for any number of bits up to the
-   maximum available, and keeps the most significant data leftmost in
-   the bit sequence.
-
-   To perform the sub-second math, simply take the first (most
-   significant/leftmost) N bits of subsec and divide it by 2^N.  Take
-   for example:
-
-   1.  To parse the first 16 bits, extract that value as an integer and
-       divide it by 65536 (2 to the 16th).
+   A shared knowledge scheme is not required by the UUID to provide
+   uniqueness guarantees.  Implementations MAY implement a shared
+   knowledge scheme as they see fit to extend the uniqueness guaranteed
+   by this UUIDs in this specification and [RFC4122].
 
-   2.  If these 16 bits are 0101 0101 0101 0101, then treating that as
-       an integer gives 0x5555 or 21845 in decimal, and dividing by
-       65536 gives 0.3333282
+   So instead just decide if you need 100% guarantee of uniqueness.  If
+   so, implement shared knowledge approach (see section below)
```

Comment:
lines preceding and following this are a bit too informal.  Basically search for uses of "you", "your", "we" or "our" and reword.